package com.memksim.todo.utils.consts

const val UNKNOWN_ERROR = "произошла непредвиденная ошибка"
const val ADD_TASK_ERROR = "не удалось сохранить задачу"
const val UPDATE_TASK_ERROR = "не удалось обновить задачу"
const val REMOVE_TASK_ERROR = "не удалось удалить задачу"
const val LOAD_TASK_ERROR = "не удалось загрузить данные"